src/sources/opentelemetry/mod.rs

```diff
@@ -136,13 +140,21 @@ impl SourceConfig for OpentelemetryConfig {
         let http_tls_settings = MaybeTlsSettings::from_config(&self.http.tls, true)?;
         let protocol = http_tls_settings.http_protocol_name();
         let bytes_received = register!(BytesReceived::from(Protocol::from(protocol)));
-        let filters = build_warp_filter(acknowledgements, log_namespace, cx.out, bytes_received);
+        let filters = build_warp_filter(
+            acknowledgements,
+            log_namespace,
+            cx.out,
+            bytes_received,
+            events_received,
+        );
         let http_source =
             run_http_server(self.http.address, http_tls_settings, filters, cx.shutdown);
 
         Ok(join(grpc_source, http_source).map(|_| Ok(())).boxed())
     }
 
+    // TODO: appropriately handle "severity" meaning across both "severity_text" and "severity_number",
+    // as both are optional and can be converted to/from.
```

Comment:
Do we already know when this is going to happen?

I'm generally an advocate of not adding TODOs without some understanding of how/when that will be addressed, often a gh issue number in the comment is ideal if it's a known enhancement we want to make in the future. Otherwise in my experience the TODOs get lost in the dust or we have to scramble people together to re-create context.

Reply:
We currently only have semantic meaning of "severity" attached to text style severities, but we've yet to use that meaning for any sink. Since we don't have a need, I didn't want to determine how to do that here, without a larger discussion or thought on the direction it needs to go.